.github/workflows/ci.yaml

```diff
@@ -67,6 +70,26 @@ jobs:
           push: false
           pull: true
           file: docker/Dockerfile
+          platforms: linux/amd64,linux/arm64
+          tags: debos
+          cache-from: type=local,src=/tmp/.build-cache
+          cache-to: type=local,dest=/tmp/.build-cache,mode=max
+
+      # WORKAROUND:
```

Comment:
In future, could we work around this with a registry in the runner ? We should really run tests on both docker images :-)

Reply:
Yes this would resolve the issue. It also makes it really easy for a reviewer to test a PR's content within a container. With a proper retention policy the containers can be cleaned up afterwards. Alternatively a hook on PR close/merge could do the cleanup.

Reply:
I created https://github.com/go-debos/debos/issues/514 to track the unit tests. I don't think we should block on that for merging this PR.